proxy/server/server.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2016-2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package server
```

Comment:
ok.

Reply:
That's a fair point, although the proxy already has an http server -- the docker registry. The proxy is by design a docker registry, and I fear opening up a common http server alongside it will quickly give way to feature creep.

If this is to be an opt-in feature (which I think it should), let's restrict the scope.

Reply:
actrully it is supposed there is a common http server port in proxy before i want to add preheat feature.
proxy is designed as a portal of kraken. i believe that there will be more feature implemented from proxy in the future. like healthz or prometheus interface even other customized feature.